src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -0,0 +1,41 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.settings
+
+import com.intellij.openapi.ui.ComboBox
+import com.intellij.ui.components.JBLabel
+import com.intellij.util.ui.FormBuilder
+import org.jdesktop.swingx.JXTitledSeparator
+import javax.swing.JCheckBox
+import javax.swing.JComponent
+import javax.swing.JPanel
+import javax.swing.JTextField
+
+/**
+ * This class displays and captures changes to the values of the Settings entries.
```

Comment:
I am not sure if this class indeed does that

Reply:
This class is responsible for settings of the plugin itself. So, any options changing the plugins UI or UX would go here, instead of the EvoSuiteSettings